app/src/main/java/com/example/secondproject/FriendProfileActivity.java

```diff
@@ -11,4 +13,14 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_friend_profile);
     }
+
+    public void launchFriendsPageActivity(View view) {
+        Intent intent = new Intent(this, FriendsPageActivity.class);
```

Comment:
Please change the name of the intent to something more indicative and as we agreed about the variables names. something like friends_page_intent will be better.